core/tests/protractor_desktop/topicAndStoryViewer.js

```diff
@@ -55,13 +55,27 @@ describe('Topic and Story viewer functionality', function() {
       language: 'English'
     };
     for (var i = 1; i <= 3; i++) {
-      await workflow.createAndPublishTwoCardExploration(
-        `Exploration TASV1 - ${i}`,
-        EXPLORATION.category,
-        EXPLORATION.objective,
-        EXPLORATION.language,
-        true
-      );
+      if (i === 1) {
+        // Set expectWelcomeModal: true.
+        await workflow.createAndPublishTwoCardExploration(
+          `Exploration TASV1 - ${i}`,
+          EXPLORATION.category,
+          EXPLORATION.objective,
+          EXPLORATION.language,
+          true,
+          true
+        );
+      } else {
+        // Set expectWelcomeModal: false.
+        await workflow.createAndPublishTwoCardExploration(
+          `Exploration TASV1 - ${i}`,
+          EXPLORATION.category,
+          EXPLORATION.objective,
+          EXPLORATION.language,
+          false,
+          true
+        );
+      }
```

Comment:
To avoid duplicating this code, you could set a variable in your if-else statement. Then use this variable when calling `createAndPublishTwoCardExploration`:

```js
var exitTutorial = false;
if (i === 1) {
  exitTutorial = true;
}
await workflow.createAndPublishTwoCardExploration(..., exitTutorial, true)
```

Reply:
Done!